meson.build

```diff
@@ -4,13 +4,18 @@ project(
   version: '1.19.1',
   meson_version: '>= 0.60',
   license: 'LGPL-2.0-or-later',
-  default_options: ['warning_level=2'])
+  default_options: [
+    'warning_level=2',
+    'force_fallback_for=libportal',
+  ],
+)
 
 ###### various directories we'll use later
 # foodir are built-in ones, foo_dir are our options
 
 prefix = get_option('prefix')
 datadir = prefix / get_option('datadir')
+libdir = prefix / get_option('libdir')
 libexecdir = prefix / get_option('libexecdir')
 sysconfdir = prefix / get_option('sysconfdir')
 localedir = prefix / get_option('localedir')
```

Comment:
FWIW, all this prefix / stuff is unnecessary. Meson appends the prefix automatically at time of use if the variable is not an absolute path.

Reply:
That doesn't seem to be the case at least in some cases. If I put `pkgconfig.generate(..., variables: {'datadir':get_option('datadir')})` for example gives me `datadir=share`.

Reply:
Good point. In that case, Meson cannot know how the value will be used. I meant more for `install_dir: get_option('localedir')`.

Reply:
Sure, but given that we might want to use them in various places I think it's better to leave it as is.

Reply:
This is also fine because:

1. you do NOT want to use 
    ```meson
    datadir = prefix / get_option('datadir')
    pkgconfig.generate(..., variables: ['datadir='+datadir])
    ```
    since that will cause your .pc file to say "prefix=/usr; datadir=/usr/share" and you actually want `datadir_for_pcfile = '${prefix}' / get_option('datadir')`

2. pkgconfig.generate will, since 0.62.0, solve this issue automatically by allowing you to use `pkgconfig.generate(..., variables: ['foobar=${datadir}/thingy'])` and automatic references for datadir will be generated for you.